network/src/error.rs

```diff
@@ -70,12 +70,6 @@ impl Display for NetworkError {
     }
 }
 
-impl NetworkError {
```

Comment:
Can we keep this? Could be usable as we improve error handling, and I'm worried people will try to work around the absence of this accessor rather than implementing it.

Other change lgtm

Reply:
@bothra90 done!

Reply:
I was going to say that same thing :)